rkt/rkt.go

```diff
@@ -27,6 +27,7 @@ import (
 	"github.com/coreos/rkt/rkt/config"
 	rktflag "github.com/coreos/rkt/rkt/flag"
 	"github.com/spf13/cobra"
+	_ "github.com/spf13/cobra/doc" // transitive dependency for man page generation
```

Comment:
I use godep v60.

Wait, I found something, godep explicitely ignores `+ignore` tags, if I use `+doc` tags in the `_gen.go` files, godep is happy.